maxfw/utils/image_utils.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import division
+import sys
+from PIL import Image
+import collections
+
+from . import image_functions as F
+
+if sys.version_info < (3, 3):
+    Sequence = collections.Sequence
+    Iterable = collections.Iterable
+else:
+    Sequence = collections.abc.Sequence
+    Iterable = collections.abc.Iterable
+
+
+_pil_interpolation_to_str = {
```

Comment:
Is this used anywhere?

Reply:
Thanks, removed this. It's not used anymore since the last update.